docs/asciidoc/modules/ROOT/pages/overview/apoc.trigger/apoc.trigger.install.adoc

```diff
@@ -0,0 +1,47 @@
+////
+This file is generated by DocsTest, so don't change it!
+////
+
+= apoc.trigger.install
+:description: This section contains reference documentation for the apoc.trigger.install procedure.
+
+label:procedure[] label:apoc-core[]
+
+[.emphasis]
+CALL apoc.trigger.install(databaseName, name, statement, selector, config) | add a trigger kernelTransaction under a name, in the kernelTransaction you can use $createdNodes, $deletedNodes etc., the selector is {phase:'before/after/rollback/afterAsync'} returns previous and new trigger information. Takes in an optional configuration.
```

Comment:
```suggestion
`CALL apoc.trigger.install(databaseName, name, statement, selector, config) | eventually adds a trigger for a given database which is invoked when successful write transactions occur.
```

Reply:
I wonder if information such as "in the kernelTransaction you can use $createdNodes, $deletedNodes etc., the selector is {phase:'before/after/rollback/afterAsync'} returns previous and new trigger information. Takes in an optional configuration" wouldn't be clearer if we put it further down in the document, where the inputs are described.

We could use the same kind of table we use for https://neo4j.com/labs/apoc/4.4/overview/apoc.periodic/apoc.periodic.iterate/ which IMO is a slight improvement.

Reply:
Good point, I "inherited" this quite confusing documentation from the `apoc.trigger.add`. I will update it.